.gitattributes

```diff
@@ -3,3 +3,14 @@ Dockerfile* linguist-language=Dockerfile
 vendor.mod linguist-language=Go-Module
 vendor.sum linguist-language=Go-Checksums
 *.bat text eol=crlf
+*.sh text eol=lf
+
+*.env -text
+
+*.png binary
+*.ico binary
+*.gif binary
+*.jpg binary
+*.sum binary
```

Comment:
Aren't Go's `*.sum` files plain text where plain `\n` is fine/sane/correct, even?

Reply:
yeah this is incorrect, I'll change the implementation to be opt-in instead of opt-out so we don't need to worry about files that are listed as binary from the vendor directory.